src/Engine/Audio/SteamAudioInterface.hpp

```diff
@@ -9,122 +9,120 @@
 /// Interface to set up and send data to/from Steam Audio from the rest of the engine.
 class SteamAudioInterface {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') class 'SteamAudioInterface' does not have a copy constructor which is recommended since the class contains a pointer to allocated memory. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AnoCopyConstructor)